src/modules/layout/components/Toolbar.css

```diff
@@ -0,0 +1,28 @@
+@import url('../../common/constants/dimensions.css');
+
+.toolbar {
+  display: flex;
+  width: 75px;
+  box-sizing: border-box;
+  flex-direction: column;
+  align-items: center;
+  padding: 15px 0;
+}
+
+.toolbar > * {
+  opacity: 0.5;
+  font-size: 1.5rem;
+  transition: 0.2s opacity;
```

Comment:
In my opinion opacity 0.5 looks a bit too much "inactive", just like the News Button when there are no news. Maybe we should also use the same value for the transition time and the tooltip delay-show.

Reply:
Still open for discussion.